src/main/kotlin/org/rust/lang/core/grammar/rust.bnf

```diff
@@ -1124,6 +1122,6 @@ external lteq   ::= collapse LTEQ LT EQ       { name = "<=" }
 external oror   ::= collapse OROR OR OR       { name = "||" }
 
 // Trailing commas are allowed
-private meta comma_separated_list ::= <<param>> ( ',' <<param>> ) * ','?
+private meta comma_separated_list ::= <<fn_param>> ( ',' <<fn_param>> ) * ','?
```

Comment:
Ok I get it. Grammar-Kit gets crazy when it's fed with `<< param >>` instead of `<<param>>` (notice whitespaces). Would you like me to fix all such occurrences in grammar in this PR?